microns_utils/__init__.py

```diff
@@ -0,0 +1,3 @@
+from . import version_utils
+
+__version__ = version_utils.get_package_version(repo='microns-utils', package='microns-utils', url_suffix='version.py')
```

Comment:
Import `\_\_version__` from `version.py` and don't run it automatically in \_\_init__. Can still have a convenience function with the latest version check available in version_utils or somewhere if wanted, just not something that's automatically run on import.